### PR TITLE
express routes created for authentication and adding projects

### PR DESCRIPTION
**Pull request by ayushkumar2808**

### CODE REVIEW

File: server/routes/auth.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                res.json({message:"saved successfully"})
+    
+            })
+            .catch(err=>{
+                console.log(err)
+            })
+        })
+
+    })
+    .catch(err=>{
```
Comment (charupraneeth) [inline server/routes/auth.js:43]: Can we improve error handling by communicating to the client that there was an error?

File: server/models/postModel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    link:{
+type:String,
+required:true
+    },
+    postedBy:{
+        type:id,
+        ref:"User"
+    }
+})
+mongoose.model("Post",postSchema)
```
Comment (charupraneeth) [inline server/models/postModel.js]: Can we export this model and import it wherever needed?

**Comment by charupraneeth** [comment]
LGTM

**Review by PatelNeelMahesh — APPROVED** [review]
You can merge pull request.

**Pull request merged**